src/app/domains/MediaLibrary/components/MediaLibraryItemSimpleView/MediaLibraryItemSimpleView.js

```diff
@@ -0,0 +1,128 @@
+import React, { cloneElement, useEffect, useState } from 'react'
+import { useHistory } from 'react-router-dom'
+import { deleteData } from 'app/services/Firestore'
+import COLLECTIONS from 'app/constants/collection'
+import {
+  Avatar,
+  Card,
+  Dropdown,
+  Image,
+  Menu,
+  message,
+  Popconfirm,
+  Typography
+} from 'antd'
+import { FormSimpleViewEdit } from 'domains/Form/components'
+import { ROUTES_PATHS } from 'app/constants'
+import { styles } from './MediaLibraryItemSimpleView.styles'
+import { FileOutlined, MoreOutlined } from '@ant-design/icons'
+import { Box, Col, Row } from '@qonsoll/react-design'
+import PropTypes from 'prop-types'
+import firebase from 'firebase/app'
+// import PropTypes from 'prop-types'
+// import { useTranslation } from 'react-i18next'
+const { Meta } = Card
+const { Text } = Typography
+
+function MediaLibraryItemSimpleView(props) {
+  const { imageUrl, name } = props
+  // const { ADDITIONAL_DESTRUCTURING_HERE } = user
+  // [ADDITIONAL HOOKS]
+  // const { t } = useTranslation('translation')
+  // const { currentLanguage } = t
+  const history = useHistory()
+  // [COMPONENT STATE HOOKS]
+  const [visible, setVisible] = useState(false)
+  const [confirmLoading, setConfirmLoading] = useState(false)
+
+  // [COMPUTED PROPERTIES]
+  const handleCancel = () => {
+    setVisible(false)
+  }
+
+  const showPopconfirm = () => {
+    setVisible(true)
+  }
+
+  // const handleDelete = async () => {
+  //   setConfirmLoading(true)
```

Comment:
Make this code work, or remove unnecessary comment al all... (currently it's optional, but on the later phases, it will be obligatory)

Reply:
done